recipes/tensap/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set version = "1.1" %}
+
+package:
+  name: tensap
+  version: {{ version }}
+
+source:
+  url: https://github.com/anthony-nouy/tensap/archive/v{{ version }}.tar.gz
+  sha256: 4074eb9cb76460114c4e54a0558a948a6464a71dee53b384a0bf734f156986d3
+
+build:
+  number: 0
+  # tensorflow is only available on linux/py<3.9 for now
+  skip: true  # [not linux]
```

Comment:
It looks like this package is pure Python. If the missing dependency is the only reason for these `skip`s you can change them to `noarch: python`, and the lack of the availability of the dependency will keep the package from installing in the other environments. We can ignore the failing CI builds.

Reply:
Thanks for the hint, I applied your suggestion!